### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by CuDecker** [CuDecker/MI-449-SS17-740-css-sass-intro-8dVLYn#1]
@KatieMFritz Can you take a look at this? It's [hosted here](https://CuDecker.github.io/MI-449-SS17-740-css-sass-intro-8dVLYn/) and meets the following criteria:

- [x] All components in [this HTML](https://gist.github.com/egillespie/15433761e640077f93660b4555a3d88b) are styled using Sass
- [x] Use four colors from a [random color palette](https://coolors.co/) (just copy the hex values into your Sass file) and link to your palette in the HTML
- [x] Font sizes are all determined by one base font size variable, even in `input` and `textarea` elements
- [x] All text uses the same font family, even in `input` and `textarea` elements
- [x] Elements shaped like a box (`img`, `input[type="text"]`, `textarea`) have the same `border-radius`
- [x] At least one built-in Sass function is used to style a component
- [x] Variables are defined in their own Sass file
- [x] Sass files are saved in a `source` directory and CSS files are saved in a `compiled` directory
- [x] A source map for your component library is included in the project

Soooo i don't know why but it doesn't seem to be the same exact thing that's saved on my computer. (Aka why the image is going outside the body) It's not like that on my end, so I'm looking into it. But everything else should be there.

**Comment by CuDecker** [comment]
When I use git add -A .

This is a warning I'm getting. What does this mean?

$ git add -A .
warning: LF will be replaced by CRLF in compiled/css/style.css.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in compiled/css/style.css.map.
The file will have its original line endings in your working directory.

**Comment by KatieMFritz** [comment]
> Soooo i don't know why but it doesn't seem to be the same exact thing that's saved on my computer. (Aka why the image is going outside the body) It's not like that on my end, so I'm looking into it. But everything else should be there.

It looks okay to me!

* * *
> When I use git add -A .
> 
> This is a warning I'm getting. What does this mean?
> 
> $ git add -A .
> warning: LF will be replaced by CRLF in compiled/css/style.css.
> The file will have its original line endings in your working directory.
> warning: LF will be replaced by CRLF in compiled/css/style.css.map.
> The file will have its original line endings in your working directory.

Good question! It's about how whitespace is saved. You don't need to worry about it. [Here's the long explanation.](http://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types)

* * *
Nice job on this! 💥 🌯 

### `input` and `textarea` look different from the rest

>  Font sizes are all determined by one base font size variable, even in input and textarea elements

>  All text uses the same font family, even in input and textarea elements

So, `input` and `textarea` are weird and use the browser default styles unless you EXPLICITLY override their `font-family` and `font-size` properties. Check out your page - your `input` and `textarea` look different from the rest, right? 👀 

### Consistent Spacing
Your line breaks in your main Sass file are a little irregular. Make sure you are using an empty line between each chunk of CSS. 🙂 

* * *
After you’ve made your changes and pushed them to GitHub and your hosted site, give it a once-over to make sure it looks right, then comment back here and I’ll take another look.

Thanks! :rocket:

**Comment by CuDecker** [comment]
I've fixed the issues. But I'm having issues getting the hosting site actually show the changes.

You can view the changes on github here
https://github.com/CuDecker/MI-449-SS17-740-css-sass-intro-8dVLYn/blob/master/source/scss/style.scss

I'll keep trying to get it to update on the host site.

**Comment by KatieMFritz** [comment]
I think it's working on the host site! I see new styles for `input` and `textarea` at least. Maybe there was a caching problem or something? 😕 

Anyway, ship it :shipit: !

![beautiful](https://cloud.githubusercontent.com/assets/7085184/25409982/5904e9fa-29e2-11e7-9ff4-fb3d3f1638a2.jpg)

**Comment by CuDecker** [comment]
Yeah! I think Stuart showed me a shortcut to refresh the cache and after I did that it totally worked. 

:D Thank youuu!
I really enjoyed working with you through everything! 👍

**Comment by KatieMFritz** [comment]
I enjoyed working with you too! You've done a great job sharpening your web dev skillz. Keep in touch! :smile:
